src/test/java/org/apache/commons/lang3/builder/ReflectionDiffBuilderTest.java

```diff
@@ -48,10 +48,14 @@
         private final Object[] objectArrayField = {null};
         private static int staticField;
         private transient String transientField;
+        @DiffExclude
+        private String annotatedField = "a";
+        private String excludedField = "a";
```

Comment:
You should test a non-transient null field.

Reply:
There is already a testcase `test_transient_field_difference` that validates this, there is no specific or new logic for transient fields that get excluded.